docs/user_guide/model_management.md

```diff
@@ -224,6 +224,8 @@ request is received under
 'config.pbtxt' is modified in place. The swap file is not part of the model
 configuration, so its presence in the model directory may be detected as a new file
 and cause the model to fully reload when only an update is expected.
+  * The model instance reused might carry a different name than the one provided
```

Comment:
```suggestion
  * When a model configuration is reloaded existing model instances will be checked against the new configuration and be reused if possible. If a model instance is reused it will retain its original name.
```